cmd/background-heal-ops.go

```diff
@@ -50,6 +50,15 @@ type healRoutine struct {
 
 // Add a new task in the tasks queue
 func (h *healRoutine) queueHealTask(task healTask) {
+	opts := globalHealConfig
+
+	// Wait and proceed if there are active requests
+	waitForLowHTTPReq(int32(globalEndpoints.NEndpoints()), opts.Sleep)
```

Comment:
Why is this better than to use the per-disk values? This will only make it slower in bigger setups with more disks/server.

IMO the wait should *only* be at the storage level and not at the API level.

Reply:
This is better because it's already used @klauspost we have double sleeps and also crawler sleeps a very high i.e 15secs max not a good idea IMO.

It is also good that we heal faster on smaller setups to make sure that disks heal fast otherwise we wait way longer and cause failures eventually with data loss.